### PR TITLE
docs: Fix broken links

### PR DESCRIPTION
**Pull request by tara-det-ai**
## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/tutorials/pachyderm-cat-dog.rst [hunk 1]
```diff
@@ -9,9 +9,8 @@
 .. meta::
    :description: Follow along with this batch inferencing tutorial to see how to leverage Determined and Pachyderm together to streamline complex tasks.
 
-In this guide, we'll help you create a simple batch inferencing project in `Pachyderm
-<https://docs.pachyderm.com/latest/learn/glossary/pipeline/>`__, train your model using a Determined
```
Comment (MikhailKardash) [inline docs/tutorials/pachyderm-cat-dog.rst:13]: This link seems to work for me. 
See:

(tutorials/pachyderm-cat-dog: line   12) redirect  https://docs.pachyderm.com/latest/learn/glossary/pipeline/ - permanently to https://docs.pachyderm.com/products/mldm/latest/learn/glossary/pipeline/

File: docs/tutorials/pachyderm-cat-dog.rst [hunk 2]
```diff
@@ -23,8 +22,8 @@ cluster, and then use the model in an inferencing pipeline.
 ************
 
 After completing the steps in this tutorial, you will have a a fully-built batch inferencing
-`pipeline <https://docs.pachyderm.com/latest/learn/glossary/pipeline/>`__ with results and you will
```
Comment (MikhailKardash) [inline docs/tutorials/pachyderm-cat-dog.rst:26]: see above comment

File: docs/tutorials/pachyderm-cat-dog.rst [hunk 3]
```diff
@@ -130,9 +130,8 @@ You are now ready to create a project repo.
  Create Repos in Pachyderm for Training Data
 *********************************************
 
-To manage our training data effectively, we'll first need to create `repos
-<https://docs.pachyderm.com/latest/learn/basic-concepts/#basic-concepts-repositories-repo>`_ for
```
Comment (MikhailKardash) [inline docs/tutorials/pachyderm-cat-dog.rst:134]: This link also still works:  https://docs.pachyderm.com/products/mldm/latest/learn/basic-concepts/#basic-concepts-repositories-repo

File: docs/tutorials/pachyderm-cat-dog.rst (hunk 1 above)
Comment (tara-det-ai) [reply docs/tutorials/pachyderm-cat-dog.rst:13]: this one should not have been there

File: docs/tutorials/pachyderm-cat-dog.rst (hunk 2 above)
Comment (tara-det-ai) [reply docs/tutorials/pachyderm-cat-dog.rst:26]: leaving this one out because of some upcoming changes

**Review by MikhailKardash — COMMENTED** [review]
The real broken link is:

1:release-notes.rst:2482: [broken] https://0pointer.de/blog/projects/socket-activation.html: HTTPSConnectionPool(host='0pointer.de', port=443): Max retries exceeded with url: /blog/projects/socket-activation.html 

I don't see a fix for it in this PR

File: docs/tutorials/pachyderm-cat-dog.rst (hunk 3 above)
Comment (tara-det-ai) [reply docs/tutorials/pachyderm-cat-dog.rst:134]: removed it due to its awkwardness

**Review by MikhailKardash — APPROVED** [review]

**Pull request merged**